contracts/allocators/LUSDAllocator.sol

```diff
@@ -180,14 +182,23 @@ contract LUSDAllocator is Ownable {
     /* ======== DEPENDENCIES ======== */
 
     using SafeERC20 for IERC20;
+    using SafeERC20 for IWETH;
     using SafeMath for uint256;
 
+    event Deposit(address indexed dst, uint amount);
+
     /* ======== STATE VARIABLES ======== */
     IStabilityPool immutable lusdStabilityPool;
```

Comment:
Can probably just define these immutable addresses before hand

Reply:
The only reason this was setup was to allow for mocking contracts via testing - see an example here https://github.com/OlympusDAO/olympus-contracts/pull/144/files#diff-e56f7d0dde3ebdd882dd5f4894b877355ef468a0fd94040e541252eb1f8de144R310

Reply:
Let me know if this crazy constructor causes issues (for deployment or anything?).   If so I can make another constructor that fills in these default values

Reply:
> The only reason this was setup was to allow for mocking contracts via testing - see an example here [#144 (files)](https://github.com/OlympusDAO/olympus-contracts/pull/144/files#diff-e56f7d0dde3ebdd882dd5f4894b877355ef468a0fd94040e541252eb1f8de144R310)

this is good reasoning imo. should keep what you have.